exercises/reverse-string/reverse-string.spec.js

```diff
@@ -0,0 +1,32 @@
+var reverseString = require('./reverse-string');
+
+describe('ReverseString', function () {
+  it('empty string', function () {
+    var expected = '';
+    var actual = reverseString('');
+    expect(actual).toEqual(expected);  
+  });
+  it('a word', function () {
+    var expected = 'tac';  
```

Comment:
For this test, the canonical data tests reversing `robot` instead of `cat`.

Please change this test to match the canonical one.